src/services/db/GitFileSystemService.ts

```diff
@@ -9,20 +9,22 @@ import {
   Result,
```

Comment:
![Screenshot 2023-08-11 at 3 48 01 PM](https://github.com/isomerpages/isomercms-backend/assets/5507822/427266ec-d959-4246-8214-fd93083a93bb)
This should not be an user facing error (though this error should not happen on prod)

Reply:
Hmm currently we return a `GitFileSystemError` which RepoService will throw and be caught by the middleware, which shows this error toast to the user. Do we want to just throw the error without any message?

Reply:
Adjusted the messages in 61eee9b7183edd03221f8d24330a2978047ffc29.